astropy/wcs/utils.py

```diff
@@ -4,6 +4,8 @@
 
 from astropy import units as u
 from astropy.utils.misc import unbroadcast
+import copy
+from scipy.optimize import least_squares
```

Comment:
This introduces a non-optional dependency on the scipy package. Can you move the import into the function below?